src/components/ConnectButton/index.tsx

```diff
@@ -78,6 +78,7 @@ const ConnectButton = (props) => (
       }
     }}
     variant="contained"
+    testId="connect-btn"
```

Comment:
I'm not sure this will work because it's possible to have two `ConnectButton` components at the same time (welcome page + header). I suggest you to find the actual place where we use this component and set the id there.

Reply:
Found each button and put a different Id to each one